### PR TITLE
Chunk sync script

### PR DESCRIPTION
**Pull request by snipe**
This just chunks the query for the count sync script so that we can avoid memory issues in very large datasets.

### CODE REVIEW

File: app/Console/Commands/SyncAssetCounters.php [hunk 1]
```diff
@@ -39,33 +39,39 @@ public function __construct()
...
-                }
-                $bar->finish();
-
-                foreach ($output['info'] as $key => $output_text) {
-                    $this->info($output_text);
-                }
-
-                $time_elapsed_secs = microtime(true) - $start;
-                $this->info('Sync executed in '.$time_elapsed_secs.' seconds');
+            ->withTrashed()->chunk(100, function ($assets) use ($bar, $start) {
```
Comment (uberbrady) [inline app/Console/Commands/SyncAssetCounters.php]: I think you don't need the `$start` here; I don't see it used anywhere in the function, only after the `chunk()` command.

**Review by uberbrady — APPROVED** [review]
This looks great and absolutely makes sense to me - loading up 10's of thousands of assets into a big giant collection is eventually going to set things on fire. I did notice one small nit, which I pointed out, but everything looks great here.

File: app/Console/Commands/SyncAssetCounters.php (hunk 1 above)
Comment (snipe) [reply app/Console/Commands/SyncAssetCounters.php]: You're right - that was leftover from some earlier refactoring

**Review by uberbrady — APPROVED** [review]
This looks great! Thanks for the quick fix :)

**Pull request merged**